Test/integration-tests/cypress/e2e/examples/payment.cy.js

```diff
@@ -376,8 +376,8 @@ describe('Payments', function () {
 
         })
     })
-
-    it('CC Pay by link', function () {
+    //skipping this test
+    it.skip('CC Pay by link', function () {
```

Comment:
Why are you skipping the tests?

Reply:
@emicha Due to a version change in Magento, these tests are not functioning as expected. Taimur is currently debugging the issue. Once resolved, we will incorporate these tests as well. But overall functionality is working fine.

Reply:
@emicha The tests have been fixed and reverted.